content/2_Bronze/Intro_Graphs.problems.json

```
@@ -22,8 +22,7 @@
       "isStarred": false,
       "tags": ["Coloring"],
       "solutionMetadata": {
-        "kind": "USACO",
-        "usacoId": "916"
+        "kind": "internal",
```

```suggestion
        "kind": "internal"
```
should fix the build